lib/run_loop/life_cycle/core_simulator.rb

```diff
@@ -453,11 +453,19 @@ def wait_for_device_state(target_state)
       end
 
       # @!visibility private
+      def csstore_file
+        @csstore_file ||= File.join(device_data_dir, 'Library', 'Caches', 'com.apple.LaunchServices-134.csstore')
+      end
+
+      # @!visibility private
       def reset_app_sandbox_internal_shared
         [app_documents_dir, app_tmp_dir].each do |dir|
           FileUtils.rm_rf dir
           FileUtils.mkdir dir
         end
+        if File.exist? csstore_file
+          FileUtils.rm_f csstore_file
+        end
```

Comment:
For iOS 7 Simulators, this file will probably never exist or exist in a different location.

The branch on exist? is not necessary - the FileUtils.rm_f will suppress an error/warning.

+100 for using `File.exist?` vs. the deprecated `File.exists?`.